config/database.php

```diff
@@ -116,22 +105,40 @@
     |--------------------------------------------------------------------------
     |
     | Redis is an open source, fast, and advanced key-value store that also
-    | provides a richer set of commands than a typical key-value systems
+    | provides a richer body of commands than a typical key-value system
     | such as APC or Memcached. Laravel makes it easy to dig right in.
     |
     */
 
     'redis' => [
 
+        'client' => env('REDIS_CLIENT', 'predis'),
+
         'cluster' => false,
 
+        // Not sure if fine to replace cluster option above with this new options item added prior to 5.8.
+        // 'options' => [
+        //     'cluster' => env('REDIS_CLUSTER', 'predis'),
+        //     'prefix' => env('REDIS_PREFIX', Str::slug(env('APP_NAME', 'laravel'), '_').'_database_'),
+        // ],
```

Comment:
I think updating this to the new `options` should be fine, but wanted to flag it for thoughts... 🤔

Reply:
Yep, I think this is fine!

Reply:
So @DFurnes doesn't matter that the `cluster` was previously set as a boolean `false`, but now it defaults to a string of `predis`?

Reply:
We're not using clustering, but I'd assume the default options would be fine regardless... This doesn't seem to match what's shown [in the documentation](https://laravel.com/docs/6.x/redis#configuration), though, so maybe we should follow that instead.

Reply:
Ah, that's the docs for 6.x which I didn't get to yet! Haha, I'll update and then change it in the next PR 😂 Good find.

Reply:
Also just noting those docs are missing the Redis URL option 🤔 Might need to submit a PR 😉